src/sentry/services/hybrid_cloud/user/__init__.py

```diff
@@ -110,6 +110,11 @@ class UserFilterArgs(TypedDict, total=False):
     emails: List[str]
 
 
+class UserUpdateArgs(TypedDict, total=False):
```

Comment:
I'm thinking that I like the TypedDict approach for patch style endpoints and will likely move other usages to this 👍